### DISCUSSION TITLE
Enhancement: week 6 icons

### DISCUSSION THREAD

**Discussion by SanGakuBoi** [FunkinCrew/Funkin#1653] [Ideas]
## What is your suggestion, and why should it be implemented?
I think that they should make the week 6 old bf icon pixelated too, so when it is used it doesn't look weird and out of place.
![image](https://user-images.githubusercontent.com/83707038/117141676-707d3f00-ad74-11eb-8282-57b4a564ca55.png)
something like that (even tho I'm really bad at pixel art)
:)

**Comment by TheDrawingCoder-Gamer** [comment]
Old-bf was meant as a debug option, so I think the extra time + resources required would not be worth it, considering most people will never see week 6 old-bf.

**Comment by SanGakuBoi** [comment]
yeah but, why not, wouldn't that make the easter egg much more proffesional

**Comment by steviegt6** [comment]
> yeah but, why not, wouldn't that make the easter egg much more proffesional

I don't see a reason not to PR the feature yourself if you really wanted to, but it'd be a waste of resources on the official team's part.
Also, I don't see much professionalism in easter eggs to begin with, they're inherently unprofessional, which isn't even a bad thing. Not sure what you're getting at by "[...] much more [professional]".

**Comment by ninjamuffin99** [comment]
perhaps

**Reply by libauror-a** [reply]
![image](https://user-images.githubusercontent.com/75984453/132061038-e740216b-9b41-4c22-b06d-d2dca7028a60.png)
